data/preloaded/C/basic io.txt

```diff
@@ -0,0 +1,43 @@
+/l
+C
```

Comment:
Might want to append a "Programing Language: " cos the parser currently looks out for it bfore parsing the language